### PR TITLE
react: add useRateLimits hook

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/react/src/wallet/useRateLimits.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      error: new NotFoundError(`No results found.`),
+      loading: false,
+    };
+  }
+
+  return {
+    data,
+    error: undefined,
+    loading: false,
+  };
```
Comment (cesarenaldi) [inline packages/react/src/wallet/useRateLimits.ts:72]: Is this repetition over what happen within `useReadResult` cause of the `data === null` if statement?

File: packages/react/src/wallet/useRateLimits.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+import { NotFoundError } from '../NotFoundError';
+import { useLensApolloClient } from '../helpers/arguments';
+import { ReadResult, useReadResult } from '../helpers/reads';
+import { useFragmentVariables } from '../helpers/variables';
+
+/**
+ * {@link useRateLimits} hook arguments
+ */
+export type UseRateLimitsArgs = UserCurrentRateLimitRequest;
```
[the remark this answers is not in the document]
Comment (krzysu) [reply packages/react/src/wallet/useRateLimits.ts:16]: I could default to the current user address, but what if you want to build a UI where any user can check their limits? As the API endpoint is not restricted to auth only, then I wouldn't restrict it on the SDK level either.

File: packages/react/src/wallet/useRateLimits.ts (hunk 1 above)
Comment (krzysu) [reply packages/react/src/wallet/useRateLimits.ts:72]: no, I copied the hook boilerplate from useProfile, but didn't notice that in useProfile `null` is a valid response and in  rateLimits it's not. I removed it now. Good catch!

File: packages/react/src/wallet/useRateLimits.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (krzysu) [reply packages/react/src/wallet/useRateLimits.ts:16]: actually, other Wallet hooks, like useProfilesManaged or useOwnedHandles, they don't default to auth user, they just accept the address, like the API query does, so I'd leave it here the same for consistency. Then it's up to a builder to use it how they want.

**Pull request merged**